pkg/pillar/dpcreconciler/linux.go

```diff
@@ -180,8 +181,9 @@ type LinuxDpcReconciler struct {
 	prevArgs     Args
 	prevStatus   ReconcileStatus
 	radioSilence types.RadioSilence
-}
 
+	kubeClusterMode bool
```

Comment:
I would suggest making `kubeClusterMode` an exported field and having it initialized [here](https://github.com/lf-edge/eve/blob/master/pkg/pillar/cmd/nim/nim.go#L175-L183). This will avoid reading `/run/eve-virt-type` on non-EVE host when running DPC Reconciler [unit tests](https://github.com/lf-edge/eve/blob/master/pkg/pillar/dpcreconciler/linux_test.go). In all current unit tests you can leave this unset (i.e. `false`). Later we may add new unit tests targeting kubevirt network config, where `KubeClusterMode` would be set to `true`. But the main point is to call `IsKubeCluster()` outside of DPC Reconciler.

Reply:
Thanks, I see it has been done - marking as resolved.